include/cutlass/relatively_equal.h

```diff
@@ -56,9 +56,7 @@ template <typename T>
 CUTLASS_HOST_DEVICE
 bool relatively_equal_float(T a, T b, T epsilon, T nonzero_floor) {
 
-#if defined (CUTLASS_ENABLE_SYCL)
-  using cutlass::abs;
-#elif defined(__CUDACC_RTC__)
+#if defined(__CUDACC_RTC__)
   using cuda::std::abs;
 #else
   using std::abs;
```

Comment:
Will `std::abs` work for custom data types like bf16 and fp16s ?

Reply:
Something like that. This function is calling `abs` without the `std::` qualifier. So if the type is supported by std::abs, the compiler will use that. Otherwise, it will check if the type has its own `abs` function accessible from the function namespace.

So, `std::abs` doesn't work with `bfloat16_t` or `half_t` types, but both contain their own `abs` implementation.